djstripe/models/base.py

```diff
@@ -382,6 +389,46 @@ def _create_from_stripe_object(
 
         return instance
 
+    @classmethod
+    def get_or_create(cls, **kwargs):
```

Comment:
shouldnt this be "get, retrieve, or create" ?

i.e. creation of a new object should only occur if it is not found locally nor found on stripe.

https://github.com/dj-stripe/dj-stripe/pull/1235 helps with the "_get_or_retrieve" part of that problem.

Reply:
#1235 was merged, so this PR should probably take advantage of the new `_get_or_retrieve`.

The new function could still be called "get_or_create" if it is clearly documented as also retrieving the record if it already exists.